packages/cypher-builder/src/expressions/operations/math.test.ts

```diff
@@ -18,28 +18,75 @@
-        labels: ["Movie"],
+        const queryResult = matchQuery.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`
+                  "MATCH (this0:\`Movie\`)
+                  WHERE this0.released = 10 + $param0
+                  RETURN this0"
+              `);
+
+        expect(queryResult.params).toMatchInlineSnapshot(`
+                  Object {
+                    "param0": 2000,
+                  }
+              `);
+    });
+
+    test("plus", () => {
+        const add = Cypher.plus(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(add).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 + 3"`);
+    });
+
+    test("minus", () => {
+        const subtract = Cypher.minus(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(subtract).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 - 3"`);
+    });
+
+    test("divide", () => {
+        const divide = Cypher.divide(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(divide).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 / 3"`);
+    });
+
+    test("multiply", () => {
+        const multiply = Cypher.multiply(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(multiply).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 * 3"`);
     });
 
-    const matchQuery = new Cypher.Match(movieNode)
-        .where(Cypher.eq(movieNode.property("released"), Cypher.plus(new Cypher.Literal(10), yearParam)))
-        .return(movieNode);
+    test("mod", () => {
+        const mod = Cypher.mod(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(mod).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 % 3"`);
+    });
 
-    const queryResult = matchQuery.build();
-    expect(queryResult.cypher).toMatchInlineSnapshot(`
-        "MATCH (this0:\`Movie\`)
-        WHERE this0.released = 10 + $param0
-        RETURN this0"
-    `);
+    test("pow", () => {
+        const pow = Cypher.pow(new Cypher.Literal(10), new Cypher.Literal(3));
+        const { cypher } = new TestClause(pow).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 ^ 3"`);
+    });
 
-    expect(queryResult.params).toMatchInlineSnapshot(`
-        Object {
-          "param0": 2000,
-        }
-    `);
+    test("complex expression", () => {
+        const add = Cypher.plus(new Cypher.Literal(10), new Cypher.Literal(3), new Cypher.Literal(5));
+        const minus = Cypher.minus(add, new Cypher.Literal(3));
+        const divide = Cypher.divide(minus, new Cypher.Literal(2));
+        const multiply = Cypher.multiply(divide, new Cypher.Literal(3));
+        const pow = Cypher.pow(multiply, new Cypher.Literal(3));
+        const { cypher } = new TestClause(pow).build();
+        expect(cypher).toMatchInlineSnapshot(`"10 + 3 + 5 - 3 / 2 * 3 ^ 3"`);
```

Comment:
yeah, we definitely need to tackle grouping here. I'll add it to the trello card